src/reaper_osara.cpp

```diff
@@ -3351,6 +3356,11 @@ void CALLBACK handleWinEvent(HWINEVENTHOOK hook, DWORD event, HWND hwnd, LONG ob
 			// screen scraping, which causes spurious messages to be reported.
 			annotateAccRole(hwnd, ROLE_SYSTEM_PANE);
 		}
+
+		bool focusIsMidiEditorEventsListView = isMidiEditorEventsListView(hwnd);
+		if (focusIsMidiEditorEventsListView) {
```

Comment:
nit: The variable doesn't really help here and is only used once.
```suggestion
		if (isMidiEditorEventsListView(hwnd)) {
```